kahuna/public/js/search/query.html

```diff
@@ -62,7 +62,14 @@
                                gu:preset-dates="searchQuery.sinceOptions"
                                gu:first-day="1">
                 </gu-date-range>
-            <li>
+            </li>
+
+            <li class="search__filter-item">
+                <select class="search__filter"
+                        ng:model="searchQuery.filter.orderBy"
+                        ng:options="sortOrder.value as sortOrder.label for sortOrder in searchQuery.orderByOptions">
+                </select>
+            </li>
```

Comment:
> It applies to the date range

It sort of does. Maybe this has been implemented the wrong way - there should be an option in the date picker to specify the field to use, then another option to specify the ordering. That way you can search for all images taken today, for example. Rather than all images uploaded today, but then order by date taken (which could be before the upload time).

I think https://github.com/guardian/grid/issues/946 is also required to make the dates on the thumbnails relevant for the order/filter you're using.

> could it live closer to the results, by the result count

Arguably, the result count would have to be sticky for this to happen.

> In terms of the label - you could try adding "order by ____" in the options to save space?

Don't like...

![screen shot 2015-07-28 at 10 34 39](https://cloud.githubusercontent.com/assets/836140/8928137/4a434642-3514-11e5-8c9f-2c82e16dc0f7.png)